Alllow rgba images by skipping the alpha channel (for now, until
the next version has full blown rgba support).

Index: edisplay/edisplay.cc
===================================================================
--- ./edisplay/edisplay.cc	(revision 241)
+++ ./edisplay/edisplay.cc	(working copy)
@@ -366,7 +366,7 @@
   if (image->spp == 1 && image->bps == 8)
     colorspace_gray8_to_rgb8 (*image);
   
-  if (image->bps != 8 || image->spp != 3) {
+  if (image->bps != 8 || (image->spp != 3 && image->spp != 4)) {
     std::cerr << "Unsupported colorspace. bps: " << image->bps
 	      << ", spp: " << image->spp << std::endl;
     std::cerr << "If possible please send a test image to rene@exactcode.de."
@@ -378,9 +378,10 @@
   unsigned char* data = (unsigned char*) malloc (image->w*image->h*4);
   unsigned char* src_ptr = image->data;
   unsigned char* dest_ptr = data;
-  
+
+  const int spp = image->spp; 
   for (int y=0; y < image->h; ++y)
-    for (int x=0; x < image->w; ++x, dest_ptr +=4, src_ptr += 3) {
+    for (int x=0; x < image->w; ++x, dest_ptr +=4, src_ptr += spp) {
 #ifndef __BIG_ENDIAN__
       dest_ptr[0] = src_ptr[2];
       dest_ptr[1] = src_ptr[1];
